wormhole-connect/src/store/transferInput.ts

```diff
@@ -71,6 +71,7 @@ export interface TransferInputState {
   amount?: amount.Amount;
   receiveAmount: DataWrapper<string>;
   route?: string;
+  isExecutorRoute?: boolean; // TODO Replace this when we have a similar prop in route object
```

Comment:
If we had an IS_EXECUTOR_ROUTE static property on the Route object, would we be able to remove this property from the store?